doc/dnf5_plugins/config-manager.8.rst

```diff
@@ -0,0 +1,156 @@
+
+      Variables are loaded from multiple directories. The list of directory paths is taken from the `varsdir` option.
+      The `setvar` command stores variables to the last directory in the list (by default "/etc/dnf/vars"). Variables from this directory are read last and override the values of the same variables defined in previous directories. In other words, the last directory has the highest priority.
+
+      Note:
+      The variables "releasever_major" and "releasever_minor" are read-only. Their values are generated from the "releasever" variable.
+
+``unsetvar <variable>+``
+    | Removes variables.
+
+      Variables are loaded from multiple directories. The list of directory paths is taken from the `varsdir` option.
+      The `unsetvar` command removes variables from the last directory in the list (by default "/etc/dnf/vars"). So, the variable may still exist in another directory in the list (for example, the default distribution value).
+
+      Note:
+      The variables "releasever_major" and "releasever_minor" are generated automatically and connot be removed.
+
+Options
+=======
+
+``--add-or-replace``
+    | Allow adding or replacing the repository with the same ID in the existing configuration file.
+
+``--create-missing-dir``
+    | Allow creation of missing directories.
+
+``--from-repofile=REPO_CONFIGURATION_FILE_URL``
+    | Specifies the source configuration file with the new repositories.
+
+``--id=REPO_ID``
+    | Set ID for newly created repository.
+
+``--overwrite``
+    | Allow replacing the existing repository configuration file by new one.
+
+``--save-filename=FILENAME``
+    | Set the name of the new repository configuration file. The ".repo" extension is added if it is missing.
+
+``--set=REPO_OPTION=VALUE``
+    | Set option in newly created repository.
+
+
+Examples
+========
+
+``dnf5 config-manager addrepo --from-repofile=http://example.com/some/additional.repo``
+    | Download "additional.repo", test it and put it in repository configuration directory.
+
+``dnf5 config-manager addrepo --set=baseurl=http://example.com/different/repo``
+    | Create new repo file with "http://example.com/different/repo" as `baseurl` and enable it. The repository ID and target file name is generated from `baseurl`.
+
+``dnf5 config-manager addrepo --set=baseurl=http://example.com/different/rep --id=example --set=enabled=0``
+    | Create new repo file with "http://example.com/different/repo" as `baseurl`. Set repository ID to "example" and disable it.
+
+``dnf5 config-manager setopt repoid1.enabled=1 repoid2.enabled=0``
+    | Sets override to enable repository identified by "repoid1" and disable repository identified by "repoid2".
+
+``dnf5 config-manager setopt repo1.proxy=http://proxy.example.com:3128/ repo2.proxy=http://proxy.example.com:3128/``
+    | Sets override for `proxy` option in repositories with repository IDs "repo1" and "repo2".
+
+``dnf5 config-manager setopt *-debuginfo.gpgcheck=0``
```

Comment:
Maybe put single quotes around these wildcards, or else the shell would expand them.